.github/workflows/main.yml

```diff
@@ -22,12 +22,18 @@ jobs:
       with:
         node-version: ${{ matrix.node-version }}
     - uses: bahmutov/npm-install@v1
+    - run: yarn playwright install-deps chromium
     - run: yarn build
     - run: yarn build:es
     - run: yarn build:all
     - run: yarn lint
+      if: matrix.os == 'ubuntu-latest'
```

Comment:
something preventing lint from running on mac & windows?

Reply:
No you just had a comment in there saying "note that we do linting here, but not on Windows / MacOS (no need to)" but you might mean in the specific mac and windows files instead

Reply:
Ah okay. I think that comment would be outdated - we had a problem where eslint was failing on windows only so I wanted to make sure it passes on all platforms

Reply:
Ok, I'll run on all platforms and see what happens